dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -1001,6 +1003,145 @@ def _get_percentile(self, percentiles):
             quantiles[percentiles == 50] = median_value
         return quantiles.tolist()
 
+    @property
+    def median_absolute_deviation(self):
+        """
+        Get median absolute deviation.
+
+        :return: median absolute deviation
+        """
+        if not self._has_histogram:
+            return np.nan
+
+        bin_counts = self._stored_histogram['histogram']['bin_counts']
+        bin_edges = self._stored_histogram['histogram']['bin_edges']
+
+        # normalize bin counts
+        bin_counts = bin_counts.astype(float)
+        normalized_bin_counts = bin_counts / np.sum(bin_counts)
+
+        # calculate deviations from median
+        median = self._get_percentile([50])[0]
```

Comment:
I think this can be modified after #389 is merged (using self.median). Also, maybe there should be an validate check in options to not compute this if median is disabled. Thoughts?

Reply:
good point. would still store the value so not repeatedly called.

Reply:
I've added an if statement to process this